src/mscorlib/src/System/Text/Normalization.Windows.cs

```diff
@@ -49,6 +56,13 @@ internal static bool IsNormalized(String strInput, NormalizationForm normForm)
 
         internal static String Normalize(String strInput, NormalizationForm normForm)
         {
+            if (GlobalizationMode.Invariant)
+            {
+                // In Invariant mode we assume all characters are normalized. 
```

Comment:
This mean that there will be a bunch of URL parsing tests failing when this is turned on because of they depend on normalization. Have you tried running all corefx tests with this (not just the globalization ones) - what is failing?